api/meter.go

```diff
@@ -567,16 +629,20 @@ func (r *relayMeter) StartDataLoader(ctx context.Context) {
 			r.Logger.WithFields(logger.Fields{"error": err}).Warn("Error setting timespan for data loader")
 			return
 		}
+
 		r.Logger.WithFields(logger.Fields{"from": from, "to": to}).Info("Starting data loader...")
+
 		if err := r.loadData(from, to); err != nil {
 			r.Logger.WithFields(logger.Fields{"error": err}).Warn("Error setting timespan for data loader")
 		}
 	}
 
 	r.Logger.WithFields(logger.Fields{"maxArchiveAge": maxPastDays}).Info("Running initial data loader iteration...")
 	load(maxPastDays)
+
```

Comment:
Are these blank lines added by the linter?

Reply:
I added them to improve readability. I know this is a matter of opinion but I find logic much more readable when it's broken up logically with some whitespace. Not a sticking point for me though, can remove if you want.

Reply:
I think we can keep the extra blank lines if they are added intentionally to improve readability.